cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -113,6 +116,8 @@ pub use unincluded_segment::{Ancestor, UsedBandwidth};
 pub use pallet::*;
 
 const LOG_TARGET: &str = "parachain-system";
+const INBOUND_MESSAGES_COLLECTION_SIZE_LIMIT: usize = 1024 * 1024; // 1 MiB
```

Comment:
Docs please. Also also what considerations on why this was set to 1 MiB.

Reply:
Changed this to 1/3 of the max block PoV size